lib/next-amp/src/components/AmpForm.tsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as React from 'react';
+import {FunctionComponent, FormHTMLAttributes} from 'react';
+import {AmpIncludeCustomElement} from '../../src-gen/';
+
+/**
+ * Renders an amp-form element.
+ *
+ * @param {
+  FormHTMLAttributes<HTMLFormElement>,
```

Comment:
Should each line here be prefixed with "*" or must it be removed to be parsed correctly? (Minor)

Reply:
Removed all JSDOC param declarations. This is redundant when using typescript